server/graphql/v2/input/ExpenseCreate.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphQLString, GraphQLInputObjectType, GraphQLList, GraphQLNonNull } from 'graphql';
+import { ExpenseType } from '../enum/ExpenseType';
+import { ExpenseAttachmentCreate } from './ExpenseAttachmentCreate';
+import { PayoutMethodInput } from './PayoutMethodInput';
+import { AccountInput } from './AccountInput';
+
+/**
+ * Input type to use as the type for the expense input in createExpense mutation.
+ */
+const ExpenseCreate = new GraphQLInputObjectType({
```

Comment:
Shouldn't this be named `ExpenseInput`?

Reply:
The idea is to have different input types for create and edit mutations, in a similar fashion to what was done with CommentCreate and CommentEdit. For example there's no reason to accept an expense ID in the create path, and their may be some params that we don't want to allow in the edit path, splitting in two types makes that possible.

Reply:
Cool, that makes sense!